client/app/bundles/course/survey/actions/questions.js

```diff
@@ -24,27 +24,24 @@ export function addToOptionsToDelete(option) {
 }
 
 export function createSurveyQuestion(
-  courseId,
-  surveyId,
   fields,
   successMessage,
   failureMessage
 ) {
   return (dispatch) => {
-    dispatch({ type: actionTypes.CREATE_SURVEY_QUESTION_REQUEST, surveyId });
-    const endpoint = `/courses/${courseId}/surveys/${surveyId}/questions`;
-    return axios.post(endpoint, fields)
+    dispatch({ type: actionTypes.CREATE_SURVEY_QUESTION_REQUEST });
+    return CourseAPI.survey.questions.create(fields)
       .then((response) => {
         dispatch({
-          surveyId,
+          surveyId: CourseAPI.survey.responses.getSurveyId(),
```

Comment:
Yes, the `surveys` reducer does not itself utilize `surveyId`, but it delegates the updating to the `survey` reducer which does the above check for `surveyId`.